Initial.py

```diff
@@ -1,3 +1,4 @@
+#
 import boto3
 
 def lambda_handler(event, context):
```

Comment:
1. Access credentials is not provided for the ec2_client
2. This can be specified using either environmental variable which will then be exported or by using boto3 session which allows us to load profiles directly and pass credentials from the ~/.aws/credentials file